src/main/java/com/keyboardbuilder/enterprise/service/IKeyboardBuildService.java

```diff
@@ -0,0 +1,15 @@
+package com.keyboardbuilder.enterprise.service;
+
+import com.keyboardbuilder.enterprise.dto.KeyboardBuild;
+
+import java.util.List;
+
+public interface IKeyboardBuildService {
+
+    /**
+     * Keyboard Build Service handles business logic for Keyboard Build DTOs.
+     */
+
+    List<KeyboardBuild> fetchAll();
+}
+
```

Comment:
Created Service Interface and put fetchhAll() in there. Also imported the dependencies for the interface. The project was missing the Service Interface. 
Source: Lecture videos

Reply:
Good.